lib/src/pubspec_suggestors/dependency_override_updater.dart

```diff
@@ -70,13 +70,9 @@ abstract class DependencyOverrideConfig {
 
 /// Suggestor that adds overrides for over_react and react in the pubspec.
 class DependencyOverrideUpdater implements Suggestor {
-  final DependencyOverrideConfig reactOverrideConfig;
-  final DependencyOverrideConfig overReactOverrideConfig;
+  final List<DependencyOverrideConfig> overridesToMake;
 
-  DependencyOverrideUpdater({
-    this.reactOverrideConfig,
-    this.overReactOverrideConfig,
-  });
+  DependencyOverrideUpdater(this.overridesToMake);
```

Comment:
(Just throwing this here before I forget) I came across a case in the React 17 sourcegraph where the current `react17_dependency_override_update` codemod doesn't work as expected when the last line of the pubspec is not a blank line (it adds the overrides before the last line - see this [tasker-ui commit](https://github.com/Workiva/tasker-ui/pull/828/commits/0bea209e0ceb8e531384ee06f82a295f4f4c3542))

I just wanted to make you aware in case you're using the same code for this more generalized codemod!